service/controller/legacy/v29/version_bundle.go

```diff
@@ -22,6 +22,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Introduce explicit resource reservation for OS resources and container runtime.",
 				Kind:        versionbundle.KindAdded,
 			},
+			{
+				Component:   "cloudformation",
+				Description: "Setup private hosted zone for internal api/etcd load-balancers.",
+				Kind:        versionbundle.KindAdded,
```

Comment:
When we do this in the legacy controller, what are the implications of existing clusters being upgraded?

Reply:
I can't imagine any.